src/Sylius/Bundle/AdminBundle/Resources/views/Email/shipmentConfirmation.html.twig

```diff
@@ -3,11 +3,24 @@
 {% endblock %}
 
 {% block body %}
-    {% autoescape %}
-        Your order with number {{ order.number }} has been sent using {{ shipment.method }}.
-        <br/>
-        You can check it's location with the "{{ shipment.tracking }}" tracking code.
-        <br/><br/>
-        Thank you for a successful transaction.
-    {% endautoescape %}
+    {% include '@SyliusAdmin/Email/_layout.html.twig' with {'content': block('content')} %}
+{% endblock %}
+
+{% block content %}
+    <div style="text-align: center;">
+        <div>
+            Your order with number
```

Comment:
Can we introduce translations keys (without any further adjustments) to these templates? Or should it be a part of next pr?

Reply:
It is another task in the backlog